sklearn/metrics/ranking.py

```diff
@@ -47,13 +47,18 @@ def auc(x, y, reorder=False):
     Parameters
     ----------
     x : array, shape = [n]
-        x coordinates.
+        Increasing or decreasing x coordinates.
     y : array, shape = [n]
         y coordinates.
-    reorder : boolean, optional (default=False)
+    reorder : boolean, optional (default=None)
```

Comment:
can we make the default something semantically meaningful, like 'warn' or 'deprecated', please?